_includes/build/bcda_user_guide.html

```diff
@@ -1,3 +1,15 @@
 <p>
 For each request and operation, we provide example requests and example cURL commands. We will highlight any common errors or unique attributes of our API. Lastly, we will provide an example response from the API.
-</p>
\ No newline at end of file
+</p>
+<div class="ds-c-alert ds-c-alert--hide-icon">
+    <div class="ds-c-alert__body">
+      <h3 class="ds-c-alert__heading">Temporary Breaking Change to Production Requests</h3>
+      <p class="ds-c-alert__text">
+During January 2021, production users have to change how they request data. Requests must be sent to the /Group endpoint and must include a new identifier: "runouts". All the claims returned will have a service date of 12/31/2020 or earlier. Any requests to the /Group (without the "runouts" identifier) or /Patient endpoints will fail and  result in an error message. 
+</p>
+<p>
+These changes are in place to ensure that BCDA does not return any data not assigned to your organization during the transition between performance years. Please see the full details in the BCDA Google Group post: LINK.
```

Comment:
We're  going to wait to merge this PR until we have the link to the Google Group post, right?